tests/benchmarks/power_consumption/adc/src/driver_test.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/drivers/adc.h>
+#include <zephyr/kernel.h>
+#define ADC_NODE DT_ALIAS(adc0)
+static const struct device *adc = DEVICE_DT_GET(ADC_NODE);
+#define LOG_LEVEL LOG_LEVEL_DBG
```

Comment:
not used?

Reply:
Removed all LOG_LEVEL definitions - they are unnecessary now.